industry_restaurant/demo/appointment_type.xml

```diff
@@ -2,6 +2,14 @@
 <odoo noupdate="1">
     <record id="appointment_type_1" model="appointment.type">
         <field name="is_published" eval="True"/>
-        <field name="resource_ids" model="appointment.resource" eval="[(6, 0, obj().search(['|', ('name', 'ilike', 'Patio'), ('name', 'ilike', 'Floor')]).ids)]"/>
+        <field name="image_1920" type="base64" file="industry_restaurant/static/src/binary/appointment_resource/table1.jpg"/>
+        <field name="location_id" ref="base.main_partner"/>
+        <field name="assign_method">time_auto_assign</field>
+        <field name="category">recurring</field>
+        <field name="avatars_display">hide</field>
+        <field name="max_schedule_days">30</field>
+        <field name="slot_creation_interval">0.25</field>
```

Comment:
Why do you need to re-define all these fields? Some are already defined in data/, no need to define the exact same thing here (example: `<field name="assign_method">time_auto_assign</field>`)